pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.black]
+extend-exclude = '''
+/(
+    segtypes/n64/pygfxd/*
```

Comment:
What is `segtypes/n64/pygfxd/` ?

(also if this is meant to be introduced I think you meant
```suggestion
    segtypes/n64/pygfxd
```
the last / is already after, and `*` isn't a wildcard (this is a regex) but afaik this can match a directory and it will exclude everything under it)

Reply:
It's been removed now